### PR TITLE
Trying to eliminate all possible ways to write CLUSTER_SEEDS

### LINKED ISSUE #134: Quoting `CLUSTER_SEEDS` list randomly breaks Akka.Cluster formation
Affects version 0.5.1 and previous

Cluster formation will be broken, randomly as a result of environment variable randomization, if we quote `CLUSTER_SEEDS`:

```
apiVersion: v1
kind: Namespace
metadata:
  name: lighthouse-sbr2
---
apiVersion: v1
kind: Service
metadata:
  name: lighthouse
  namespace: lighthouse-sbr2
  labels:
    app: lighthouse
spec:
  clusterIP: None
  ports:
  - port: 4053
  selector:
    app: lighthouse
---
apiVersion: apps/v1
kind: StatefulSet
metadata:
  namespace: lighthouse-sbr2
  name: lighthouse
  labels:
    app: lighthouse
spec:
  serviceName: lighthouse
  replicas: 2
  selector:
    matchLabels:
      app: lighthouse
  template:
    metadata:
      labels:
        app: lighthouse
    spec:
      terminationGracePeriodSeconds: 35
      containers:
      - name: lighthouse
        image: arkatufus/lighthouse:1.5.2
        lifecycle:
          preStop:
            exec:
              command: ["/bin/sh", "-c", "pbm 127.0.0.1:9110 cluster leave"]
        env:
        - name: ACTORSYSTEM
          value: AkkaTrader
        - name: POD_NAME
          valueFrom:
            fieldRef:
              fieldPath: metadata.name
        - name: CLUSTER_IP
          value: "$(POD_NAME).lighthouse"
        - name: CLUSTER_SEEDS
          value: "akka.tcp://$(ACTORSYSTEM)@lighthouse-0.lighthouse:4053,akka.tcp://$(ACTORSYSTEM)@lighthouse-1.lighthouse:4053,akka.tcp://$(ACTORSYSTEM)@lighthouse-2.lighthouse:4053"
        - name: AKKA__CLUSTER__DOWNING_PROVIDER_CLASS
          value: "Akka.Cluster.SBR.SplitBrainResolverProvider, Akka.Cluster"
        - name: AKKA__CLUSTER__SPLIT_BRAIN_RESOLVER__ACTIVE_STRATEGY
          value: "keep-majority"
        livenessProbe:
          tcpSocket:
            port: 4053
        ports:
        - containerPort: 4053
          protocol: TCP
```

If I unquote `CLUSTER_SEEDS` then cluster formation works fine:

```
apiVersion: v1
kind: Namespace
metadata:
  name: lighthouse-sbr2
---
apiVersion: v1
kind: Service
metadata:
  name: lighthouse
  namespace: lighthouse-sbr2
  labels:
    app: lighthouse
spec:
  clusterIP: None
  ports:
  - port: 4053
  selector:
    app: lighthouse
---
apiVersion: apps/v1
kind: StatefulSet
metadata:
  namespace: lighthouse-sbr2
  name: lighthouse
  labels:
    app: lighthouse
spec:
  serviceName: lighthouse
  replicas: 2
  selector:
    matchLabels:
      app: lighthouse
  template:
    metadata:
      labels:
        app: lighthouse
    spec:
      terminationGracePeriodSeconds: 35
      containers:
      - name: lighthouse
        image: arkatufus/lighthouse:1.5.2
        lifecycle:
          preStop:
            exec:
              command: ["/bin/sh", "-c", "pbm 127.0.0.1:9110 cluster leave"]
        env:
        - name: ACTORSYSTEM
          value: AkkaTrader
        - name: POD_NAME
          valueFrom:
            fieldRef:
              fieldPath: metadata.name
        - name: CLUSTER_IP
          value: "$(POD_NAME).lighthouse"
        - name: CLUSTER_SEEDS
          value: akka.tcp://$(ACTORSYSTEM)@lighthouse-0.lighthouse:4053,akka.tcp://$(ACTORSYSTEM)@lighthouse-1.lighthouse:4053,akka.tcp://$(ACTORSYSTEM)@lighthouse-2.lighthouse:4053
        - name: AKKA__CLUSTER__DOWNING_PROVIDER_CLASS
          value: "Akka.Cluster.SBR.SplitBrainResolverProvider, Akka.Cluster"
        - name: AKKA__CLUSTER__SPLIT_BRAIN_RESOLVER__ACTIVE_STRATEGY
          value: "keep-majority"
        livenessProbe:
          tcpSocket:
            port: 4053
        ports:
        - containerPort: 4053
          protocol: TCP
```

### PR DESCRIPTION
**Pull request by Arkatufus**
Closes #134

Really need to find a way to stop yak shaving this... I hope this is comprehensive enough.
NOTE: I dropped checks for triple quotes, its just not worth it.

### CODE REVIEW

File: src/Akka.Bootstrap.Docker.Tests/DockerBootstrapSpecs.cs [hunk 1]
```diff
@@ -30,7 +42,33 @@ public void ShouldStartIfValidSeedNodesIfSupplied(string seedNodes)
                 var myConfig = ConfigurationFactory.Empty.BootstrapFromDocker();
                 myConfig.HasPath("akka.cluster.seed-nodes").Should().BeTrue();
                 var seeds = myConfig.GetStringList("akka.cluster.seed-nodes").Select(x => x.Trim());
-                seeds.Should().BeEquivalentTo(seedNodes.Split(",").Select(x => x.Trim()));
+                seeds.Should().BeEquivalentTo(seedNodes.Replace("\"", "").Split(",").Select(x => x.Trim()));
```
Comment (Aaronontheweb) [inline src/Akka.Bootstrap.Docker.Tests/DockerBootstrapSpecs.cs:45]: Can you add one more test case that does a hard coded apples to apples comparison of seed nodes quoted different ways, where the expected values are constant and not modified via trim / unquoting in any way? Just want a sanity check in there should anyone ever move some of this stuff around again in the future.

**Review by Aaronontheweb — CHANGES_REQUESTED** [review]
Need one minor test case added, but otherwise should be good to go

File: src/Akka.Bootstrap.Docker.Tests/DockerBootstrapSpecs.cs (hunk 1 above)
Comment (Arkatufus) [reply src/Akka.Bootstrap.Docker.Tests/DockerBootstrapSpecs.cs:45]: Sure, give me a minute

Comment (Arkatufus) [reply src/Akka.Bootstrap.Docker.Tests/DockerBootstrapSpecs.cs:45]: Done

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Pull request merged**